cairo_programs/get_env.json

```diff
@@ -1,7 +1,7 @@
 {
   "attributes": [],
   "builtins": ["output"],
-  "compiler_version": "0.13.1",
+  "compiler_version": "0.12.0",
```

Comment:
Looks like you don’t use the project env. it won’t create any issue here and in any case with Kakarot migratiin I’ve fully revamped it, but fyi this shouldn’t be here

Reply:
This is just my version of cairo lang which is not the same as yours locally no? I've tried to update to 0.13.2 (last) and it gives me the right compiler version here

Reply:
this is supposed to be fixed for the project here https://github.com/kkrt-labs/keth/blob/main/pyproject.toml#L10 meaning that you didn't run the project correct version. Though again, it doesn't change the result

Reply:
Ah yes I see what you mean, I've just rerun and push using poetry so that this is clean :)